spec/fixtures/type-verifications.csv

```diff
@@ -0,0 +1,53 @@
+application/msword,PageDescription
```

Comment:
Would you mind renaming this file to pcdm-type-verifications.csv, to distinguish from dc type?